support-frontend/assets/pages/digital-subscription-landing/components/paymentSelection/helpers/paymentSelection.jsx

```diff
@@ -91,6 +93,34 @@ const BILLING_PERIOD = {
   },
 };
 
+const matchesQueryParam = promotion =>
+  getQueryParameter(promoQueryParam) === promotion.promoCode;
+const matchesABVariant = (promotion: Promotion, abVariant: Option<string>) => {
+  if (!abVariant) {
+    return false;
+  }
+
+  if (abVariant === 'one-for-one') {
+    return promotion.promoCode.toLowerCase() === 'one-for-one';
+  }
+
+  return promotion.promoCode === 'DK0NT24WG';
+};
+
+// When the 'one for one vs 3 months' ab test is complete this function can
+// be deleted and we can use the getAppliedPromo function from promotions.js
```

Comment:
Is there a way to make this reusable (assuming that's even a good idea) so it isn't temporary?

Reply:
I looked at that but it would mean passing a function into the `getAppliedPromo` function which seemed to me to add too much complexity for the benefit it would bring